### PR TITLE
Improving html tables and columns

### PR DESCRIPTION
**Pull request by stefanvignir**
This is a re-submission after Houndtore everything down last time (https://github.com/activeadmin/activeadmin/pull/3808)

Those are SCSS modifications, improving the display of tables and columns.

### CODE REVIEW

File: app/assets/stylesheets/active_admin/components/_columns.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .columns {
+  @include clearfix;
   margin-bottom: 10px;
 }
+.column {
```
Comment (timoschilling) [inline app/assets/stylesheets/active_admin/components/_columns.scss]: Rule declaration should be followed by an empty line

**Comment by timoschilling** [comment]
Please change the following thinks:
1. Rebase it, otherwise I can't merge
2. Apply the 2 last comments
3. Squash it into one commit
4. Please provide some before / after screenshots

HINT: you don't need to create a new PR on a change, you can simply force push to your PR branch.

**Comment by stefanvignir** [comment]
Okay. Expect a new pull request in about a week or so.

**Comment by timoschilling** [comment]
Any updates?

**Comment by timoschilling** [comment]
@stefanvignir Any updates?

**Comment by timoschilling** [comment]
I'm closing this PR while you don't response and @javierjulio make a bigger CSS change in #3862.

**Pull request closed without merging**